extensions-core/histogram/src/main/java/io/druid/query/aggregation/histogram/sql/QuantileSqlAggregator.java

```diff
@@ -77,37 +81,58 @@ public Aggregation toDruidAggregation(
       return null;
     }
 
+    final AggregatorFactory aggregatorFactory;
+    final String histogramName = String.format("%s:agg", name);
     final RexNode probabilityArg = Expressions.fromFieldAccess(
         rowSignature,
         project,
         aggregateCall.getArgList().get(1)
     );
     final float probability = ((Number) RexLiteral.value(probabilityArg)).floatValue();
 
-    final AggregatorFactory aggregatorFactory;
-    final String histogramName = String.format("%s:agg", name);
+    final int resolution;
+    if (aggregateCall.getArgList().size() >= 3) {
```

Comment:
That's true, although now if we ever add a 4th argument this line doesn't have to change :)